server/routes/index.js

```diff
@@ -31,4 +32,5 @@ export default (app) => {
   app.use('/api/v1/area', area);
   app.use('/api/v1/content-property', contentProperty);
   app.use('/api/v1/report-property', reportProperty);
+  app.use('/api/v1/next-payments', nextPayment);
```

Comment:
I noticed that we used singular naming for all routes here. Can we do the same for next payments too 🙏🏽 

```suggestion
  app.use('/api/v1/next-payment', nextPayment);
```